src/test/resources/expected-classes/interfaces/BarBar.java.txt

```diff
@@ -0,0 +1,13 @@
+package com.kobylynskyi.graphql.interfaces;
+
+
+@javax.annotation.Generated(
+    value = "com.kobylynskyi.graphql.codegen.GraphQLCodegen",
+    date = "2020-12-31T23:59:59-0500"
+)
+public interface BarBar extends Bar{
```

Comment:
maybe need add a Space

Reply:
Thanks for pointing this out. I will make a note and do this in the next PR.